src/Button/sizes.scss

```diff
@@ -3,18 +3,30 @@
   font-weight: var(--font-weight-semibold);
   min-height: rem(24px);
   border-radius: rem(12px);
+  .nds-button-content {
+    margin: rem(3px) rem(16px);
+    max-width: rem(200px);
+  }
```

Comment:
`.nds-button-content` exists to set the constraints of the label area of the button. The values here are related to `size`, so they have been moved to the appropriate size placeholders here.